app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupActivity.java

```diff
@@ -52,11 +53,17 @@ protected void initialize(int resourceId) {
         Log.d(MEETUP_KEY, "In CreateMeetup, received meetup: " + json);
         mMeetup = new Gson().fromJson(json, Meetup.class);
 
+        mBackButton = findViewById(R.id.back_button);
+        if (mBackButton == null) {
+            mBackButton = findViewById(R.id.cancel_button);
```

Comment:
It's fine because it works, but we really shouldn't attach CreateMeetupActivity as a listener for the cancel button. If it's overridden, it doesn't need to exist.